lite-rpc/src/main.rs

```diff
@@ -196,7 +208,20 @@ pub async fn start_lite_rpc(args: Config, rpc_client: Arc<RpcClient>) -> anyhow:
         let inmemory_account_storage: Arc<dyn AccountStorageInterface> =
             Arc::new(InmemoryAccountStore::new());
         const MAX_CONNECTIONS_IN_PARALLEL: usize = 10;
-        let account_service = AccountService::new(inmemory_account_storage);
+        let (account_notification_sender, _) = tokio::sync::broadcast::channel(256);
```

Comment:
comment the expected size of a single message in the channel + explain the timing characteristics of producer and consumer